packages/clerk-js/src/ui/elements/Select.tsx

```diff
@@ -241,7 +241,7 @@ export const SelectOptionList = (props: SelectOptionListProps) => {
       setFocusedIndex(-1);
       return;
     }
-    focusedItemRef.current?.scrollIntoView({ block: 'nearest' });
+    focusedItemRef.current?.scrollIntoView?.({ block: 'nearest' });
```

Comment:
I couldn't figure out how to mock this in jest.

Reply:
Can we add a short commnent here?